Harmonise README and Landing file (#2032)
alan-turing-institute/the-turing-way: book/website/welcome.md

(welcome)=
# Welcome

> *Welcome to The Turing Way guides to reproducible, ethical and collaborative data science.*

Reviewer: I think this phrasing might be confusing if we don't match the titles of the guides in the second half of the sentence.

E.g. Welcome to The Turing Way guides to Reproducible Research, Project Design, Communication, Collaboration, and Ethical Research.

Author: _The Turing Way_ is a handbook to reproducible, ethical and collaborative data science.
```suggestion
_The Turing Way_ is a community-led project to reproducible, ethical and collaborative data science.
```
I guess my reason to edit this line was to avoid the assumption that The Turing Way is only a handbook. Does this editing on your original work better?

Reviewer: I'm curious to understand better what you mean by 'only a handbook'? 

'project to' doesn't work in my opinion. It would need to be 'project which aims to enable' or 'project where the goal is to teach' or something like that.

Author: The Turing Way is a book as well as a community.
I agree that the sentence needs more update, I edited it, but then again it is longer than what you suggested.

Author: what could be the best filler for `<>`?

Reviewer: > The Turing Way is a book as well as a community-led project that does more than just writing chapters in a handbook.

Thanks for the clarification. I agree.
However, I think it would be reasonable in this first line to welcome people to the guide/handbook and then to flow on with more detail about the project in a broader sense in the following sentences.

I would suggest:
```suggestion
*Welcome to The Turing Way handbook to reproducible, ethical and collaborative data science.*
```
so long as it is quickly followed by 'the TR project is open XYZ and community driven, and aims to foster/engage/create a community etc etc'

Author: I like the suggestion on the following line clarifying that. I am committing the suggested change.

> _The Turing Way_ is an open source, open collaboration and community-driven project.

Reviewer: I am disappointed to see this revert to the previous form (after #1961).
In #1961 I expanded and clarified the terms, with the hope to make this sentence both easier to read and reduce jargon. 
I also flipped the sentence around so that we could avoid having to hyphenate open-source (https://github.com/alan-turing-institute/the-turing-way/pull/1961#issuecomment-884598869).

Author: Hey Danny, at the time of editing this, the branch was not rebased.
My aim was to come back and edit to make sure that I don't completely change your phrasing - while also not make the intro lines linking out. Please bare with me as I work through this.

Reviewer: I can understand the desire to avoid links so early on.

I distinguish between open source (where you can *see* the source code) and open licenced (where you're allowed to *use* the source code), but perhaps that conceptual distinction is not one that actually occurs too much.

I think if there is room for 'community-driven' then it's a great addition - it is something which is important for the project that previously wasn't noted.

To fix the grammar it could be flipped around like so:
```suggestion
_The Turing Way_ project is open source, open collaboration, and community-driven.
```

_[I think I might have been tired/grumpy when I wrote the previous comment - I like this version better now than when I wrote my previous comment! :flushed:]_

We involve and support a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.

> Our goal is to provide all the information that researchers and data scientists in academia, industry and the public sector need at the start of their projects to ensure that they are easy to reproduce at the end.

Reviewer: I would suggest restructuring this sentence for easier reading (as was done in #1961) 
```suggestion
Our goal is to provide all the information that researchers and data scientists in academia, industry and the public sector need to ensure that the projects they work on are easy to reproduce and reuse.
```

> ```{note}

Reviewer: Love the idea of having this as a note. That should break it up a little, and fits the tone.

_The Turing Way_ is not meant to be read from start to finish. 

> Start with a concept, tool or method that you need now, in your current work. 

Reviewer: Shame to lose the buffet reference, but I really like this new wording ❤️

Author: Thanks, Danny. I like the buffet reference too, but I want to avoid jargon in open research generated within Europe and the USA context, that may not make a lot of sense for global readers. I know it is too picky, but I hope this rephrasing doesn't lose the meaning.

Reviewer: It keeps the meaning beautifully. And you're right - less jargon / 'in jokes' the better! I hadn't thought about it but it's possible as well that the metaphor itself might only work in certain cultures.

Author: Thank you for being patient with this PR :)

Reviewer: It feels important to get the landing page right - first impressions count! :)
Very happy to be able to help

Browse the different guides that make up the book, or use the search box to search for whatever you would like to learn about first.
```

All stakeholders, including researchers, software engineers, project leaders and funding teams, are encouraged to use _The Turing Way_ to understand their roles and responsibility of reproducibility in data science.

> You can inspect our resources on [GitHub](https://github.com/alan-turing-institute/the-turing-way), contribute to the project as described in our [contribution guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) and re-use all materials ([see the License](https://github.com/alan-turing-institute/the-turing-way/blob/master/LICENSE.md)).

Reviewer: Love the way this is worded.

Author: Thanks for the feedback, Danny. I moved the links that you had highlighted in the opening line here to avoid linking out at the beginning of the paragraph.
